docs/csharp/tutorials/intro-to-csharp/branches-and-loops-local.md

```diff
@@ -276,6 +276,42 @@ Experiment with these yourself. Try each of the following:
 When you're done, let's move on to write some code yourself to
 use what you've learned.
 
+## Created nested loops
+
+A `while`, `do` or `for` loop can be nested inside another loop to create a matrix using the combination of each item in the outer loop with each item in the inner loop. Let's do that to build a set of alpha-numeric pairs to represent rows and columns.
+
+One `for` loop can generate the rows:
+
+```csharp
+for (int row = 1; row < 11; row++)
+{
+    Console.WriteLine($"The row is {row}");
+}
+```
+
+Another loop can generate the columns:
+
+```csharp
+for (char column = 'a'; column < 'k'; column++)
+{
+    Console.WriteLine($"The column is {column}");
+}
+```
+
+You can nest one loop inside the other to form pairs:
+
+```csharp
+for (int row = 1; row < 11; row++)
+{
+    for (char column = 'a'; column < 'k'; column++)
+    {
+        Console.WriteLine($"The cell is ({row}, {column})");
+    }
+}
+```
+
+You can see that the outer out loop increments once for each iteration of the inner loop. Reverse the row and column nesting, and see the changes for yourself.
```

Comment:
> the outer out loop increments once for each iteration of the inner loop

That doesn't sound correct. I've read it as `row` is incremented once for each iteration of the inner loop, that is, for each increment of `column`. 

Did you want to say:
```suggestion
You can see that the outer loop increments once for each full run of the inner loop. Reverse the row and column nesting, and see the changes for yourself.
```